pkg/wwan/README.md

```diff
@@ -0,0 +1,945 @@
+                     |     ip timeout: 20
+  ------------------------------------
+  Properties         |            apn: internet
+                     |        roaming: allowed
+                     |        ip type: ipv4
+  ------------------------------------
+  IPv4 configuration |         method: static
+                     |        address: 10.49.140.93
+                     |         prefix: 30
+                     |        gateway: 10.49.140.94
+                     |            dns: 213.151.233.252, 213.151.233.249
+                     |            mtu: 1500
+  ------------------------------------
+  Statistics         |       duration: 60
+                     |       attempts: 1
+                     | total-duration: 60
+```
+
+From here we can see the IP details we have been assigned by the cellular network and the network
+interface corresponding to the modem (`wwan0`).
+ModemManager does not assign IPv4 address details to the network interface in Linux by itself.
+When ModemManager is used in conjunction with NetworkManager and the cellular connection is managed
+by it, then the IPv4 address details will be collected by NetworkManager through ModemManager
+and automatically assigned to the network interface when connection is established.
+In EVE, it is the responsibility of mmagent to manage the IP configuration of the network interface.
+
+When testing a new modem on EVE manually, the IP and routing configuration needs to be applied
+using the ip command. Taking the IP config from the above as an example, we would run:
+
+```console
+# Enable network interface in Linux.
+ip link set wwan0 up
+
+# Set the IPv4 address acquired from the bearer information above.
+ip addr add 10.49.140.93/30 dev wwan0
+
+# Disable ARP.
+ip link set dev wwan0 arp off
+
+# Set MTU value acquired from network.
+ip link set dev wwan0 mtu 1500
+
+# Add a default route for the cellular network device pointing to the received gateway IP.
+ip route add default via 10.49.140.94 dev wwan0 metric 1000
+
+# Add the DNS servers reported by cellular network:
+echo 'nameserver 213.151.233.252' >> /etc/resolv.conf
+echo 'nameserver 213.151.233.249' >> /etc/resolv.conf
+```
+
+We should now have a network interface passing data successfully, we can try it out by doing ping requests:
+
+```console
+ping -4 -c 4 -I wwan0 8.8.8.8
+PING 8.8.8.8 (8.8.8.8) from 10.49.140.93 wwan0: 56(84) bytes of data.
+64 bytes from 8.8.8.8: icmp_seq=1 ttl=117 time=123 ms
+64 bytes from 8.8.8.8: icmp_seq=2 ttl=117 time=46.2 ms
+64 bytes from 8.8.8.8: icmp_seq=3 ttl=117 time=51.4 ms
+64 bytes from 8.8.8.8: icmp_seq=4 ttl=117 time=44.2 ms
+
+--- 8.8.8.8 ping statistics ---
+4 packets transmitted, 4 received, 0% packet loss, time 3005ms
+rtt min/avg/max/mdev = 44.159/66.162/122.881/32.853 ms
+```
+
+Now you have verified that your modem is compatible with EVE OS and you can try to configure
+cellular connection properly from the controller.
+
+## Frequencies and antennas
```

Comment:
This doesn't quite belong under modemmanager; move back to original file?

Reply:
Done